<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="0" total="23" passed="23">
  <reporter-output>
  </reporter-output>
  <suite name="Surefire suite" duration-ms="952" started-at="2016-10-28T14:40:16Z" finished-at="2016-10-28T14:40:17Z">
    <groups>
    </groups>
    <test name="Surefire test" duration-ms="952" started-at="2016-10-28T14:40:16Z" finished-at="2016-10-28T14:40:17Z">
      <class name="com.yahoo.ycsb.TestUtils">
        <test-method status="PASS" signature="bytesToDoubleTooShort()" name="bytesToDoubleTooShort" duration-ms="1" started-at="2016-10-28T14:40:16Z" finished-at="2016-10-28T14:40:16Z">
          <exception class="java.lang.IllegalArgumentException">
            <message>
              <![CDATA[Byte array must be 8 bytes wide.]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.IllegalArgumentException: Byte array must be 8 bytes wide.
	at com.yahoo.ycsb.Utils.bytesToDouble(Utils.java:170)
	at com.yahoo.ycsb.TestUtils.bytesToDoubleTooShort(TestUtils.java:99)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:80)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:673)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:842)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1166)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.runWorkers(TestRunner.java:1178)
	at org.testng.TestRunner.privateRun(TestRunner.java:757)
	at org.testng.TestRunner.run(TestRunner.java:608)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1158)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1083)
	at org.testng.TestNG.run(TestNG.java:999)
	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:115)
	at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.executeMulti(TestNGDirectoryTestSuite.java:204)
	at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.execute(TestNGDirectoryTestSuite.java:107)
	at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:112)
	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:200)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:153)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:103)
]]>
            </full-stacktrace>
          </exception>
        </test-method>
        <test-method status="PASS" signature="bytesToFromDouble()" name="bytesToFromDouble" duration-ms="0" started-at="2016-10-28T14:40:16Z" finished-at="2016-10-28T14:40:16Z">
        </test-method>
        <test-method status="PASS" signature="bytesToFromLong()" name="bytesToFromLong" duration-ms="0" started-at="2016-10-28T14:40:16Z" finished-at="2016-10-28T14:40:16Z">
        </test-method>
        <test-method status="PASS" signature="bytesToLongNull()" name="bytesToLongNull" duration-ms="0" started-at="2016-10-28T14:40:16Z" finished-at="2016-10-28T14:40:16Z">
          <exception class="java.lang.NullPointerException">
            <full-stacktrace>
              <![CDATA[java.lang.NullPointerException
	at com.yahoo.ycsb.Utils.bytesToLong(Utils.java:131)
	at com.yahoo.ycsb.TestUtils.bytesToLongNull(TestUtils.java:89)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:80)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:673)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:842)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1166)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.runWorkers(TestRunner.java:1178)
	at org.testng.TestRunner.privateRun(TestRunner.java:757)
	at org.testng.TestRunner.run(TestRunner.java:608)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1158)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1083)
	at org.testng.TestNG.run(TestNG.java:999)
	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:115)
	at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.executeMulti(TestNGDirectoryTestSuite.java:204)
	at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.execute(TestNGDirectoryTestSuite.java:107)
	at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:112)
	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:200)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:153)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:103)
]]>
            </full-stacktrace>
          </exception>
        </test-method>
        <test-method status="PASS" signature="bytesToLongTooShort()" name="bytesToLongTooShort" duration-ms="1" started-at="2016-10-28T14:40:16Z" finished-at="2016-10-28T14:40:16Z">
          <exception class="java.lang.ArrayIndexOutOfBoundsException">
            <message>
              <![CDATA[7]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.ArrayIndexOutOfBoundsException: 7
	at com.yahoo.ycsb.Utils.bytesToLong(Utils.java:131)
	at com.yahoo.ycsb.TestUtils.bytesToLongTooShort(TestUtils.java:94)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:80)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:673)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:842)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1166)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.runWorkers(TestRunner.java:1178)
	at org.testng.TestRunner.privateRun(TestRunner.java:757)
	at org.testng.TestRunner.run(TestRunner.java:608)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1158)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1083)
	at org.testng.TestNG.run(TestNG.java:999)
	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:115)
	at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.executeMulti(TestNGDirectoryTestSuite.java:204)
	at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.execute(TestNGDirectoryTestSuite.java:107)
	at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:112)
	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:200)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:153)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:103)
]]>
            </full-stacktrace>
          </exception>
        </test-method>
        <test-method status="PASS" signature="jvmUtils()" name="jvmUtils" duration-ms="3" started-at="2016-10-28T14:40:16Z" finished-at="2016-10-28T14:40:16Z">
        </test-method>
      </class>
      <class name="com.yahoo.ycsb.TestByteIterator">
        <test-method status="PASS" signature="testRandomByteIterator()" name="testRandomByteIterator" duration-ms="64" started-at="2016-10-28T14:40:16Z" finished-at="2016-10-28T14:40:16Z">
        </test-method>
      </class>
      <class name="com.yahoo.ycsb.generator.TestUnixEpochTimestampGenerator">
        <test-method status="PASS" signature="ctorWithIntervalAndUnits()" name="ctorWithIntervalAndUnits" duration-ms="1" started-at="2016-10-28T14:40:17Z" finished-at="2016-10-28T14:40:17Z">
        </test-method>
        <test-method status="PASS" signature="ctorWithIntervalAndUnitsAndStart()" name="ctorWithIntervalAndUnitsAndStart" duration-ms="0" started-at="2016-10-28T14:40:17Z" finished-at="2016-10-28T14:40:17Z">
        </test-method>
        <test-method status="PASS" signature="defaultCtor()" name="defaultCtor" duration-ms="1" started-at="2016-10-28T14:40:17Z" finished-at="2016-10-28T14:40:17Z">
        </test-method>
        <test-method status="PASS" signature="variousIntervalsAndUnits()" name="variousIntervalsAndUnits" duration-ms="0" started-at="2016-10-28T14:40:17Z" finished-at="2016-10-28T14:40:17Z">
        </test-method>
      </class>
      <class name="com.yahoo.ycsb.measurements.exporter.TestMeasurementsExporter">
        <test-method status="PASS" signature="testJSONArrayMeasurementsExporter()" name="testJSONArrayMeasurementsExporter" duration-ms="196" started-at="2016-10-28T14:40:17Z" finished-at="2016-10-28T14:40:17Z">
        </test-method>
      </class>
      <class name="com.yahoo.ycsb.TestStatus">
        <test-method status="PASS" signature="testAcceptableStatus()" name="testAcceptableStatus" duration-ms="4" started-at="2016-10-28T14:40:16Z" finished-at="2016-10-28T14:40:16Z">
        </test-method>
      </class>
      <class name="com.yahoo.ycsb.generator.AcknowledgedCounterGeneratorTest">
        <test-method status="PASS" signature="testIncrementPastIntegerMaxValue()" name="testIncrementPastIntegerMaxValue" duration-ms="648" started-at="2016-10-28T14:40:16Z" finished-at="2016-10-28T14:40:17Z">
        </test-method>
      </class>
      <class name="com.yahoo.ycsb.generator.TestZipfianGenerator">
        <test-method status="PASS" signature="testMinAndMaxParameter()" name="testMinAndMaxParameter" duration-ms="6" started-at="2016-10-28T14:40:17Z" finished-at="2016-10-28T14:40:17Z">
        </test-method>
      </class>
      <class name="com.yahoo.ycsb.generator.TestIncrementingPrintableStringGenerator">
        <test-method status="PASS" signature="invalidCharacterSets()" name="invalidCharacterSets" duration-ms="1" started-at="2016-10-28T14:40:17Z" finished-at="2016-10-28T14:40:17Z">
        </test-method>
        <test-method status="PASS" signature="invalidLengths()" name="invalidLengths" duration-ms="0" started-at="2016-10-28T14:40:17Z" finished-at="2016-10-28T14:40:17Z">
        </test-method>
        <test-method status="PASS" signature="rolloverOneCharacterException()" name="rolloverOneCharacterException" duration-ms="1" started-at="2016-10-28T14:40:17Z" finished-at="2016-10-28T14:40:17Z">
        </test-method>
        <test-method status="PASS" signature="rolloverOK()" name="rolloverOK" duration-ms="0" started-at="2016-10-28T14:40:17Z" finished-at="2016-10-28T14:40:17Z">
        </test-method>
        <test-method status="PASS" signature="rolloverException()" name="rolloverException" duration-ms="0" started-at="2016-10-28T14:40:17Z" finished-at="2016-10-28T14:40:17Z">
        </test-method>
        <test-method status="PASS" signature="rolloverOneCharacterOK()" name="rolloverOneCharacterOK" duration-ms="0" started-at="2016-10-28T14:40:17Z" finished-at="2016-10-28T14:40:17Z">
        </test-method>
      </class>
      <class name="com.yahoo.ycsb.workloads.TestCoreWorkload">
        <test-method status="PASS" signature="createOperationChooser()" name="createOperationChooser" duration-ms="4" started-at="2016-10-28T14:40:17Z" finished-at="2016-10-28T14:40:17Z">
        </test-method>
        <test-method status="PASS" signature="createOperationChooserNullProperties()" name="createOperationChooserNullProperties" duration-ms="0" started-at="2016-10-28T14:40:17Z" finished-at="2016-10-28T14:40:17Z">
          <exception class="java.lang.IllegalArgumentException">
            <message>
              <![CDATA[Properties object cannot be null]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.IllegalArgumentException: Properties object cannot be null
	at com.yahoo.ycsb.workloads.CoreWorkload.createOperationGenerator(CoreWorkload.java:836)
	at com.yahoo.ycsb.workloads.TestCoreWorkload.createOperationChooserNullProperties(TestCoreWorkload.java:68)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:80)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:673)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:842)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1166)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.runWorkers(TestRunner.java:1178)
	at org.testng.TestRunner.privateRun(TestRunner.java:757)
	at org.testng.TestRunner.run(TestRunner.java:608)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1158)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1083)
	at org.testng.TestNG.run(TestNG.java:999)
	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:115)
	at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.executeMulti(TestNGDirectoryTestSuite.java:204)
	at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.execute(TestNGDirectoryTestSuite.java:107)
	at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:112)
	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:200)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:153)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:103)
]]>
            </full-stacktrace>
          </exception>
        </test-method>
      </class>
    </test>
  </suite>
</testng-results>
